script.js

```diff
@@ -0,0 +1,116 @@
+const inputJs = document.querySelector(".input"),
+    toDoNewJs = inputJs.querySelector("#todo-new"),
+    listWaitingJs = document.querySelector(".waiting"),
+    listCompleteJs = document.querySelector(".complete");
```

Comment:
변수명뒤에 Js는 무슨 뜻일까용?

```suggestion
const todoForm= document.querySelector(".input"),
    todoInput= todoForm.querySelector("#todo-input"),
    waitingList= document.querySelector(".waiting"),
    completeList= document.querySelector(".complete");
```

이렇게 해주시면 어떤 역할을 하는 변수인지 좀 더 알기 쉬울 것 같아요~

Reply:
네 더 알기 쉬운 것 같아요!! 수정했습니다 감사합니다~!

Reply:
이해하기 더 쉬워졌어요 !